lib/pbench/server/api/resources/intake_base.py

```diff
@@ -357,10 +357,7 @@ def _intake(
                         hash_md5.update(chunk)
             except OSError as exc:
                 if exc.errno == errno.ENOSPC:
-                    raise APIAbort(
-                        HTTPStatus.REQUEST_ENTITY_TOO_LARGE,
-                        f"Out of space on {tar_full_path.parent}",
-                    )
+                    raise APIAbort(HTTPStatus.REQUEST_ENTITY_TOO_LARGE, "Out of space")
```

Comment:
So ... "more and smaller PRs would be better" but ... "we should debate and make functional changes here that have no relationship to the purpose of this PR"? 😆 

If it makes you feel better I'll remove the message change, but I'm not going to *expand* the changes.

Reply:
> "we should debate and make functional changes here that have no relationship to the purpose of this PR"?

Well, but the debate/changes have a relationship to a proposed change in this PR...which brings that whole "purpose" thing into question!  🙃

> If it makes you feel better I'll remove the message change

It would have been good if it had been in a separate PR, but I've already reviewed it here, so splitting this PR will only help if doing so allows us to merge one or the other of the splits faster...which I doubt it will at this point.

> I'm not going to expand the changes.

That's fair.  But, if you end up editing doing another commit for some other reason, I do think it would be good to add a log call here and with the other related `raise`'s.